workflow-job

```diff
@@ -471,7 +495,22 @@ def merge() {
     jamesVersion = version()
     sh "docker build --tag ${images.gitPublish} -f dockerfiles/merge/Dockerfile ."
     sh "docker run ${verbose} -t --name ${containers.gitPublish} -v /origin ${images.gitPublish} ${sha1ToBuild} ${mergeBranch} ${originalBranch}"
-    merging.success();
+}
+
+def buildVolumeFromTag() {
+    git url: authorParentURL, branch: "master"
+    sh "git fetch --tags origin"
+    sh "git checkout ${tagName} -b ${mergeBranch}"
+    jamesVersion = version()
+    def dockerfile = """\
+                        FROM debian:8.11
+                        
+                        ADD . /origin
+                        WORKDIR /origin
+                        CMD echo \"Volume built\"""".stripIndent()
```

Comment:
debug?

Reply:
No, it's just the command inside the container without side effect.
It's a trick in order to build a volume